Update README.md (#86)
caskformula/homebrew-caskformula: README.md

caskformula is a tap for providing [homebrew](http://brew.sh/) formulae for software packaged as [casks](https://caskroom.github.io/).

> This document has changed.  Homebrew now includes brew-cask as part of the install. Run __brew upgrade__ or __brew cask doctor__ if you run into errors here.

Reviewer: ```suggestion
Homebrew now includes [brew-cask](https://github.com/Homebrew/homebrew-cask) as part of the install. Run `brew upgrade` or `brew cask doctor` if you run into errors below.
```
